spec/factories/goods_nomenclature_factory.rb

```diff
@@ -17,11 +17,37 @@
     validity_start_date { Date.current.ago(2.years) }
     validity_end_date   { nil }
 
+    # TODO: Put this in a trait. This forces indents on all nomenclature regardless of
+    #       what is passed to the individual factory and adds non-fun surprises for developers.
     after(:build) do |gono, evaluator|
-      FactoryBot.create(:goods_nomenclature_indent, goods_nomenclature_sid: gono.goods_nomenclature_sid,
-                                                    validity_start_date: gono.validity_start_date,
-                                                    validity_end_date: gono.validity_end_date,
-                                                    number_indents: evaluator.indents)
+      FactoryBot.create(
+        :goods_nomenclature_indent,
+        goods_nomenclature_sid: gono.goods_nomenclature_sid,
+        validity_start_date: gono.validity_start_date,
+        validity_end_date: gono.validity_end_date,
+        number_indents: evaluator.indents,
+        productline_suffix: gono.producline_suffix,
+      )
+    end
+
+    trait :with_indent do
+      # TODO: Populate this trait
+    end
```

Comment:
I guess it makes sense to have this trait,   
but it's empty and we are using it 🤔